### PR TITLE
Pdf

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: components/common/pdf-section.vue [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            pdfContainer:true,
+            importCSS:true,
+            importStyle : true,
+            // pageTitle : $('title').text(),
+            // loadCSS : '/app/css/pdf-friendly.css',
+        });	
+        // console.log(html);
+        try{
+            const captchaToken = await $recaptcha.getRecaptchaToken();
+            const res = await useAPIFetch('http://localhost:8000/api/v2017/generate-pdf',
```
Comment (sbilodeau) [inline components/common/pdf-section.vue:77]: http://localhost:8000/ ???

File: components/common/pdf-section.vue [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                const tempLink = document.createElement("a");
+                tempLink.href = url;
+                tempLink.setAttribute("download", downloadFileName.value);
+
+                // Append the <a> element to the body and click it to trigger the download
+                document.body.appendChild(tempLink);
+                tempLink.click();
+
+                // Clean up the temporary elements and URL
+                document.body.removeChild(tempLink);
```
Comment (sbilodeau) [inline components/common/pdf-section.vue:108]: No need to add and remove. At least it works for me
https://github.com/scbd/kronos-web/blob/d9c09de2618a1dec747d8b17e12d1bbb8dba5138/filters/data-converter.js#L45 (Mine missing revokeObjectURL) !
```suggestion
                tempLink.click();

                // Clean up the temporary elements and URL
```

**Review by sbilodeau — CHANGES_REQUESTED** [review]
There is a `localhost` left

File: components/common/pdf-section.vue (hunk 1 above)
Comment (blaisefonseca) [reply components/common/pdf-section.vue]: fixed

File: components/common/pdf-section.vue (hunk 2 above)
Comment (blaisefonseca) [reply components/common/pdf-section.vue]: done

**Review by sbilodeau — APPROVED** [review]

**Pull request merged**